gatsby-config.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
     {
       resolve: `gatsby-source-strapi`,
       options: {
-        apiURL: `https://civicactions-content.civicactions-content.app.civicactions.net`,
```

Comment:
@alexiscott we don't want change the api to pull from local. This will break the builds for the static site.

Reply:
I also had errors when I tried to build this. `GraphQLError: Unknown argument "filter" on field "Query.allStrapiStaffProfile"
  . Did you mean "title"`

Reply:
> @alexiscott we don't want change the api to pull from local. This will break the builds for the static site.

Thanks, good catch!